documentation/search-api.html

```diff
@@ -24,6 +24,20 @@ <h2 id="http">HTTP</h2>
 $ curl --data yql=select+%2A+from+sources+%2A+where+default+contains+%22bad%22%3B http://localhost:8080/search/
 </pre>
   </li><li>
+    The Search Container uses Jetty for HTTP.
+    Configure the <a href="reference/services-http.html#server">http server</a> - e.g. set <em>requestHeaderSize</em> to configure URL length:
```

Comment:
The `requestHeaderSize` is the limit for the overall header size, including HTTP headers. It's not only the URL length.

Reply:
fixed